adapters/repos/db/lsmkv/segment_group.go

```diff
@@ -94,19 +108,19 @@ func newSegmentGroup(dir string, logger logrus.FieldLogger,
 		// If yes, we must assume that the flush never finished, as otherwise the
 		// WAL would have been lsmkv.Deleted. Thus we must remove it.
 		potentialWALFileName := strings.TrimSuffix(entry.Name(), ".db") + ".wal"
-		ok, err := fileExists(filepath.Join(dir, potentialWALFileName))
+		ok, err := fileExists(filepath.Join(sg.dir, potentialWALFileName))
 		if err != nil {
 			return nil, errors.Wrapf(err, "check for presence of wals for segment %s",
 				entry.Name())
 		}
 
 		if ok {
-			if err := os.Remove(filepath.Join(dir, entry.Name())); err != nil {
+			if err := os.Remove(filepath.Join(sg.dir, entry.Name())); err != nil {
 				return nil, errors.Wrapf(err, "delete corrupt segment %s", entry.Name())
 			}
 
 			logger.WithField("action", "lsm_segment_init").
-				WithField("path", filepath.Join(dir, entry.Name())).
+				WithField("path", filepath.Join(sg.dir, entry.Name())).
 				WithField("wal_path", potentialWALFileName).
 				Info("Discarded (partially written) LSM segment, because an active WAL for " +
```

Comment:
Don't break strings across lines, it makes it impossible to search for error messages

Reply:
that was not changed in this PR